src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/draftorders/judgeapproval/SortKey.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.draftorders.judgeapproval;
+
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+@Setter
+@Getter
+@Data
+public class SortKey implements Comparable<SortKey> {
+    private String hearingTime;
+    private LocalDate hearingDate;
+    private LocalDateTime documentSubmittedDate;
+
+    // Constructor, Getters, and Setters
+
+    public SortKey(String hearingTime, LocalDate hearingDate, LocalDateTime documentSubmittedDate) {
+        this.hearingTime = hearingTime;
+        this.hearingDate = hearingDate;
+        this.documentSubmittedDate = documentSubmittedDate;
+    }
+
+    // Comparable implementation
```

Comment:
How about using javadoc rather than a comment to document what a method does?

Reply:
Yes sir. It's generated from ChatGPT and I didn't take much care about it sorry.